### DISCUSSION TITLE
Churches ?

### DISCUSSION THREAD

**Discussion by OcelotWalrus** [Dungeons-of-Kathallion/Bane-Of-Wargs#13] [Ideas]
Hey @lumbar527 . Any idea of what churches could do, something else than exchanging EXP?

**Comment by lumbar527** [comment]
I can't think of any that make sense.

By EXP do you mean Experience Points?

**Reply by OcelotWalrus** [reply]
Yes. I know that it don't have any sense between exchanging EXP for HP in churches, but I think there should be a way to use EXP. And since EXP kinda magic...

**Reply by OcelotWalrus** [reply]
Maybe with that, you could exchange XP with additional agility, armor protection also.

**Reply by lumbar527** [reply]
Sure!

**Reply by OcelotWalrus** [reply]
Cool!
